.github/workflows/container-release.yml

```diff
@@ -0,0 +1,37 @@
+# Build and publish containers for this release
+# see also container-build.yml
+
+name: Container Release
+
+on:
+  push:
+    tags:
+      - release-*
+
+jobs:
+  push-container:
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        GO_VERSION:
+          - "1.23.4"
+        include:
+          - dockerfile: test/ct-test-srv/Dockerfile
+            image: ghcr.io/letsencrypt/ct-test-srv
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Build
+        run: docker buildx build . --build-arg "GO_VERSION=${{ matrix.GO_VERSION }}" -f "${{ matrix.dockerfile }}" -t "${{ matrix.image }}:${{ github.ref_name }}"
```

Comment:
```suggestion
        run: docker build . --build-arg "GO_VERSION=${{ matrix.GO_VERSION }}" -f "${{ matrix.dockerfile }}" -t "${{ matrix.image }}:${{ github.ref_name }}"
```

Reply:
I believe `docker build` is usually an alias for `docker buildx build` on recent dockers at least, but I think it's better to be explicit